index.html

```diff
@@ -124,5 +124,30 @@
     <p class="navbar-cta navbar-item right" id="navbar-cta-text">We're hiring</p>
   </nav>
 
+  <div id="hiring-and-aearly-access-section">
+    <div id="join-hiring" class="card-subsection">
+      <h3> We're Hiring </h3>
+      <p> Help us redefine the calendar, reclaim our schedules, and make the time we spend together, better </p>
+      <a href="#" class="btn roles-button">See the roles</a>
```

Comment:
Don't use abbreviations for class names. They're not clear. What does this mean?